src/main/java/trackitnus/logic/parser/lesson/AddLessonCommandParser.java

```diff
@@ -43,8 +43,7 @@ public AddLessonCommand parse(String args) throws ParseException {
             ArgumentTokenizer.tokenize(args,
                 PREFIX_CODE, PREFIX_TYPE, PREFIX_DATE);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_CODE, PREFIX_TYPE, PREFIX_DATE)
-            || !argMultimap.getPreamble().isEmpty()) {
+        if (!arePrefixesPresent(argMultimap, PREFIX_CODE, PREFIX_TYPE, PREFIX_DATE)) {
```

Comment:
Why did you remove `!argMultimap.getPreamble().isEmpty()`?

Reply:
because argMultimap.getPreamble() is actually to check if index is present, which is unnecessary in this case (in fact this is leftovers of copying from AB3 commands)